Northstar.CustomServers/mod/scripts/vscripts/mp/_base_gametype_mp.gnut

```diff
@@ -420,11 +420,15 @@ void function DecideRespawnPlayer( entity player )
 
 void function RespawnAsPilot( entity player )
 {
+	if(IsValid( player )) return
```

Comment:
the IsValid check is not needed as IsAlive returns false when the entity is not valid. also it might block legit respawns